webclient/components/pt-info/single/1time-Mrow-mField/diagnosis/db/client-side/structure/master-table-of-diagnosis.js

```diff
@@ -0,0 +1,32 @@
+// For docs read webclient/docs/models.md
+import clientSideTableManage from '~/components/core/crud/manage-rows-of-table-in-client-side-orm.js'
+
+import clientSideTblPatientDiagnosis from './patient-table-of-diagnosis'
+
+const { v1: uuidv1 } = require('uuid')
+
+let count = 0
+const intUniqueID = () => ++count
+
+export default class clientSideTblMasterDiagnosis extends clientSideTableManage {
+  static entity = 'tableMasterOfDiagnosis'
+
+  static primaryKey = 'masterDiagnosisId'
+
+  static fields() {
+    return {
+      ...super.fields(),
+
+      masterDiagnosisId: this.uid(() => intUniqueID()),
+      diagnosisName: this.string(null),
+
+      parity: this.number(0),
```

Comment:
Line no: 23 -> 'parity' is not meaningful

Reply:
fixed